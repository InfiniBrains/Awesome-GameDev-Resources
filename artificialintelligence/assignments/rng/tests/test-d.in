4 20 0 999
