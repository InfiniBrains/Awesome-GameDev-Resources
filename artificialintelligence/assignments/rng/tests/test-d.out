476
140
508
444
972
503
353
707
512
61
404
46
643
660
326
506
426
69
192
417
